babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
I'm wondering if we should use `calypso-build` for this, to ensure config compatibility. Then again, I expect this will only be used for the standalone bundle, while it exists outside of the monorepo?

Reply:
My intent here was to do some cleanups before importing this project to the Calypso monorepo. Then it will use `calypso-build`. At that time, it won't be a huge upgrade of everything, but just a little change that ideally won't change the build output at all.